packages/hardhat/deploy/99_generateTsAbis.ts

```diff
@@ -68,9 +62,13 @@ const generateTsAbis: DeployFunction = async function () {
   }
   fs.writeFileSync(
     `${TARGET_DIR}deployedContracts.ts`,
-    prettier.format(`const contracts = {${fileContent}} as const; \n\n export default contracts`, {
-      parser: "typescript",
-    }),
+    prettier.format(
+      `import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract"; \n\n
+ const contracts = {${fileContent}} satisfies GenericContractsDeclaration | null ; \n\n export default contracts`,
```

Comment:
Will need to change this if we are doing https://github.com/scaffold-eth/scaffold-eth-2/pull/592#discussion_r1376002175
(if not, ignore :D)